javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,22 @@
+const class07Students = ["Anna", "Jonas", "Micah", "Noah", "Louise"];
+function addStudentToClass(studentName) {
+  // You write code here
+    if (class07Students.length>5 && studentName !== "Queen Margrethe") {
+        console.log("Cannot add more students to class 07");
+    } else if (class07Students.includes(studentName)) {
+        console.log("Student " + studentName + " is already in the class");
```

Comment:
You can use [template literals/strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for cleaner look. 
```suggestion
        console.log(`Student ${studentName} is already in the class`);
```